common/djangoapps/util/organizations_helpers.py

```diff
@@ -103,3 +104,10 @@ def organizations_enabled():
     Returns boolean indication if organizations app is enabled on not.
     """
     return settings.FEATURES.get('ORGANIZATIONS_APP', False)
+
+
+def get_course_organization(course_key):
```

Comment:
Please name this function get_course_organization_name and add a docstring to this function

Reply:
Fixed